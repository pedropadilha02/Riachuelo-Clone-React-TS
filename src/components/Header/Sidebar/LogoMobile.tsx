import styled from 'styled-components';
import { useState } from 'react';

const Logo = styled.h1`
  display: block;

  @media (max-width: 1200px) {
    opacity: 1;
    height: 60px;
    line-height: 60px;
    text-align: center;
    margin: 0;
    position: absolute;
    width: 100%;
  }
`;

export function LogoMobile() {
  return (
    <Logo>
      <img src="http://localhost:3000/assets/rchlo.svg" alt="" />
    </Logo>
  );
}
