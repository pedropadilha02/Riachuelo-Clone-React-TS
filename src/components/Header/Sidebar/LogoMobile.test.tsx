import { render } from '@testing-library/react';
import { LogoMobile } from './LogoMobile';

describe('Teste de renderização do LogoMobile', () => {
  it('O snapshot do component deve permanecer semre o mesmo', () => {
    const { container } = render(<LogoMobile />);

    expect(container).toMatchSnapshot();
  });
});
