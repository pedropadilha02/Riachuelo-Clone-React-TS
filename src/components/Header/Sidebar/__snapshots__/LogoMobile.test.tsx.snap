// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Teste de renderização do LogoMobile O snapshot do component deve permanecer semre o mesmo 1`] = `
<div>
  <h1
    class="sc-bdnxRM fbLKbm"
  >
    <img
      alt=""
      src="http://localhost:3000/assets/rchlo.svg"
    />
  </h1>
</div>
`;
